driver/main/pom.xml

```diff
@@ -48,6 +48,11 @@
             <artifactId>org.opennms.oce.datasource.jaxb</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.opennms.oce</groupId>
+            <artifactId>processor-api</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Should use ${project.version} instead of hard-coding 1.0.0-SNAPSHOT. There are other cases of this as well.